arviz/tests/test_plots.py

```diff
@@ -40,7 +40,7 @@ def test_posteriorplot(self):
 
     def test_autocorrplot(self):
         for obj in (self.short_trace, self.fit):
-            assert autocorrplot(obj).get_geometry() == (6, 6, 36)
+            assert autocorrplot(obj)[0, 0].get_geometry() == (6, 6, 36)
```

Comment:
elsewhere we're using `assert autocorrplot(obj).shape == (6, 6)`

Reply:
yeah, I like that better. Changed it to that now.